src/Umbraco.Core/Constants-SqlTemplates.cs

```diff
@@ -14,7 +14,11 @@ public static class VersionableRepository
                 public const string GetParentNode = "Umbraco.Core.VersionableRepository.GetParentNode";
                 public const string GetReservedId = "Umbraco.Core.VersionableRepository.GetReservedId";
             }
-
+            public static class RelationRepository
+            {
+                public const string DeleteByParentAll = "Umbraco.Core.RelationRepository.DeleteByParent";
```

Comment:
I can see these were in use before but that has been reverted. Any reason why we still can't use Sql templates? Even if it's just for the non-sqlce?

Reply:
I did set up the template, but I didn't know enough about how they worked with two sets of parameters so I reverted the change. Hopefully I can get some assistance with this. I think it went from ~1000ms to <40ms. But I'm not sure. There are too many queries I've looked at lately to remember. But the difference was huge.

Reply:
Original
DELETE [umbracoRelation] 
where id in ( 
SELECT r.id
  FROM [dbo].[umbracoRelation] r
  INNER JOIN  umbracoRelationType rt on rt.id = r.relType
  where r.parentId = 155928)

![image](https://user-images.githubusercontent.com/834725/102283691-ca34c680-3f97-11eb-9e4c-33da3219d799.png)

New:
  DELETE umbracoRelation
  FROM umbracoRelation
  INNER JOIN umbracoRelationType rt on rt.id = relType
  where umbracoRelation.parentId = parentId

![image](https://user-images.githubusercontent.com/834725/102283731-dc166980-3f97-11eb-97a1-e085205de678.png)